test/document_stores/test_opensearch.py

```diff
@@ -816,7 +816,7 @@ def test_bulk_write_retries_for_always_failing_insert_is_canceled(self, mocked_d
             for i in range(1000)
         ]
 
-        with patch("haystack.document_stores.elasticsearch.bulk") as mocked_bulk:
+        with patch("haystack.document_stores.opensearch.bulk") as mocked_bulk:
```

Comment:
`bulk` and `scan` were changed but the test here only covers bulk. Would it be possible without much effort to add a test for `scan` as well? Or is it covered already somewhere?

Reply:
@masci I would not block this PR for that reason but I would be a good addition in my opinion. What do you think?